src/main/java/de/hysky/skyblocker/utils/Location.java

```diff
@@ -114,6 +114,14 @@ public String id() {
         return this.id;
     }
 
+	/**
```

Comment:
I don’t think this is really needed, you can just use the getter and do a reference check. Not every location is an island either, and this also returns a boxed Boolean rather than a primitive.

Reply:
in my opinion this is better than having to write `Utils.getLocation() == Location.<someIsland>` every single time when wanting to do an island check.
And every location in this enum is a skyblock island (expect the placeholder and unknown one), and considering it uses the hypixel api to get the active island, i dont think it would support other types of locations either way

Reply:
While this is shorter than `Utils.getLocation() == Location.<someLocation>`, the latter is more explicit and easier to understand than this at a first glance, and this kind of location check is not something you write very often for it to be a huge time saver when typing anyway.